recipes/h5pickle/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "h5pickle" %}
+{% set version = "0.1" %}
+{% set sha256 = "30b5c0880ec231dccd0365716d81b5a789eb9d3599cae12a55cdfa5046f4948f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - h5py
+    - toolz
+    - cachetools
+
+test:
+  imports:
+
+about:
+  home: http://github.com/exteris/h5pickle
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
This is fine for now. To fix it long term, please see issue ( https://github.com/Exteris/h5pickle/issues/2 ).